tests/fixture/database/test_postgres.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, Integer
+from sqlalchemy.ext.declarative import declarative_base
+
+from pytest_mock_resources import create_postgres_fixture
+
+Base = declarative_base()
+
+
+class Thing(Base):
+    __tablename__ = "thing"
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+
+
+createdb_template_pg = create_postgres_fixture(Base, createdb_template="template0", session=True)
```

Comment:
documentation would be good

Reply:
this is kind of niche, it'll get "documented" in terms of the API part of the docs from the docstring, i dont think it deserves its own docs section though(?)

Reply:
sounds good